File: Examples/CSharp/AsposePDF/DocumentConversion/CGMToPDF.cs
//////////////////////////////////////////////////////////////////////////
// Copyright 2001-2014 Aspose Pty Ltd. All Rights Reserved.
//
// This file is part of Aspose.Pdf. The source code in this file
// is only intended as a supplement to the documentation, and is provided
// "as is", without warranty of any kind, either expressed or implied.
//////////////////////////////////////////////////////////////////////////
using System.IO;

using Aspose.Pdf;

namespace CSharp.AsposePdf.DocumentConversion
{
    public class CGMToPDF
    {
        public static void Run()
        {
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_AsposePdf_DocumentConversion();

            // Instantiate LoadOption object using CGMLoadOption
            Aspose.Pdf.CgmLoadOptions cgmload = new Aspose.Pdf.CgmLoadOptions();
            // instantiate Document object
            Document doc = new Document(dataDir+ "TECHDRAW.CGM", cgmload);
            // Save the resultant PDF document
            doc.Save(dataDir+ "TECHDRAW.pdf");
        }
    }
}

File: Examples/VisualBasic/AsposePDF/DocumentConversion/CGMToPDF.vb
Imports System.IO
Imports System
Imports Microsoft.VisualBasic
Imports Aspose.Pdf

Namespace AsposePDF
    Public Class CGMToPDF
        Public Shared Sub Run()
            ' ExStart:CGMToPDF
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir_AsposePdf_DocumentConversion()

            ' Instantiate LoadOption object using CGMLoadOption
            Dim cgmload As New CgmLoadOptions()
            ' Instantiate Document object
            Dim doc As New Document(dataDir & Convert.ToString("CGMToPDF.CGM"), cgmload)
            ' Save the resultant PDF document
            doc.Save(dataDir & Convert.ToString("TECHDRAW_out_.pdf"))
            ' ExEnd:CGMToPDF
        End Sub
    End Class
End Namespace